src/app/containers/MostRead/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Canonical from './Canonical';
+
+const getLocalMostReadEndpoint = (service, variant) => {
+  const localhostURL = 'http://localhost:7080';
+  const localServiceURL = `${localhostURL}/${service}`;
+
+  return variant
```

Comment:
in `index.stories` this was updated to check for whether the variant is set to `default`. Should probably do so here too.

Reply:
the logic works a little different here 🙃 

I am getting the variant value from requestContext because `serviceContext` provider doesn't actually return a variant value and the `requestContext` will default to `null`. https://github.com/bbc/simorgh/blob/latest/src/app/contexts/RequestContext/index.jsx#L85

Reply:
ah okay, thought it would return `default`. Nothing to see here
![](https://media2.giphy.com/media/XItRQJP0wai7m/giphy.gif)